samples/go/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.16.4
+RUN mkdir /app
+COPY bin/leadIntegration /app
+WORKDIR /app
+EXPOSE 8000
+CMD ["leadIntegration"]
```

Comment:
It's a good practice to always work with full path on CMD / ENTRYPOINT. Also, I strongly suggest you to use ENTRYPOINT to avoid overwrite on runtime.